packages/components/src/mobile/global-styles-context/utils.native.js

```diff
@@ -212,9 +213,9 @@ export function parseStylesVariables( styles, mappedValues, customValues ) {
 				const path = $2.split( '--' );
 				const mappedPresetValue = mappedValues[ path[ 0 ] ];
 				if ( mappedPresetValue && mappedPresetValue.slug ) {
-					const matchedValue = find( mappedPresetValue.values, {
-						slug: path[ 1 ],
-					} );
+					const matchedValue = Object.values(
```

Comment:
Will be an object:

https://github.com/WordPress/gutenberg/blob/47a6e4a8579125818e0945f086032e7e133f4909/packages/components/src/mobile/global-styles-context/utils.native.js#L287-L296

The `?? {}` adds additional safety in the case of non-existence.